### PR TITLE
Tests - Added code for generating files used for load testing using Vegeta.

### PR DESCRIPTION
**Pull request by thehenrytsai**

### CODE REVIEW

File: tests/generators/VegetaLoadGenerator.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+/**
+ * Class for generating files used for load testing using Vegeta.
+ */
+export default class VegetaLoadGenerator {
+
+  /**
+   * Creates a Create request followed by an Update request for each DID.
+   */
+  public static async generateCreateLoadFiles (uniqueDidCount: number, endpointUrl: string, absoluteFolderPath: string) {
```
Comment (sylar217) [inline tests/generators/VegetaLoadGenerator.ts]: Everything looks great, Can you please add the params defintion?

Comment (sylar217) [reply tests/generators/VegetaLoadGenerator.ts]: One more thing, how are planning to use this method?

Comment (thehenrytsai) [reply tests/generators/VegetaLoadGenerator.ts]: Parameter description added.

I am using this for load testing request throughput and operation processing throughput.

Comment (thehenrytsai) [reply tests/generators/VegetaLoadGenerator.ts]: This can be used for both local and cloud load testing.

**Pull request merged**